src/Utils/testDataUtil.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+import * as CLM from '@conversationlearner/models'
+
+export function makeEntityValues(): CLM.MemoryValue[] {
+    return [{
+        "userText": "userText",
+        "displayText": "displayText",
+        "builtinType": null,
+        "resolution": {}
+    }]
+}
+
+export function makeFilledEntity(entityId?: string): CLM.FilledEntity {
+    return {
+        entityId: entityId || CLM.ModelUtils.generateGUID(),
```

Comment:
Hmm, we already have the `uuid` library included on this repo which does a real guid.  Doesn't matter much though